<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>van benne</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nev</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="2" column="11" rowspan="4" colspan="2">
     <widget class="QListView" name="listView_3"/>
    </item>
    <item row="5" column="16">
     <widget class="QListView" name="listView_6"/>
    </item>
    <item row="1" column="10">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Hozzaad</string>
      </property>
     </widget>
    </item>
    <item row="2" column="13" rowspan="4" colspan="3">
     <widget class="QListView" name="listView_4"/>
    </item>
    <item row="1" column="11">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Cimke hozzaad</string>
      </property>
     </widget>
    </item>
    <item row="1" column="12">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="0" column="15">
     <widget class="QLineEdit" name="lineEdit_8"/>
    </item>
    <item row="1" column="15">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Feltet Hozzaad</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nyitvatartas</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7" colspan="4">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item row="2" column="6" rowspan="4" colspan="4">
     <widget class="QListView" name="listView_2"/>
    </item>
    <item row="0" column="11">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Pizzaalapar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="14">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Szallitasikoltseg</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="14">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Hozzaad</string>
      </property>
     </widget>
    </item>
    <item row="1" column="13">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="4" column="16">
     <widget class="QListView" name="listView_5"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Hozzaad</string>
      </property>
     </widget>
    </item>
    <item row="0" column="13">
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item row="2" column="0" rowspan="4" colspan="3">
     <widget class="QListView" name="listView"/>
    </item>
    <item row="2" column="10" rowspan="4">
     <widget class="QListView" name="listView_7">
      <property name="maximumSize">
       <size>
        <width>114</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1219</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionMentes"/>
    <addaction name="actionMegnyitas"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMentes">
   <property name="text">
    <string>Mentes</string>
   </property>
  </action>
  <action name="actionMegnyitas">
   <property name="text">
    <string>Megnyitas</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
